### DISCUSSION TITLE
Fensterkontakt Sensor und noch zwei weitere

### DISCUSSION THREAD

**Discussion by Trunks1982** [mak-gitdev/HA_enoceanmqtt#110] [Request]
Kann dieser Fenstergriff von Soda hinzugefügt werden?

EEP D2-06-01 , id 0410ca59

![Bildschirmfoto 2023-12-24 um 00 49 22](https://github.com/mak-gitdev/HA_enoceanmqtt/assets/62701386/090a48d6-c472-4562-b182-1c026068950c)

**Comment by mak-gitdev** [comment]
Hi @Trunks1982,

I will have a [look](https://github.com/users/mak-gitdev/projects/1/views/1?pane=issue&itemId=48367090).

**Reply by Trunks1982** [reply]
Hi, hattest du mal Zeit um nachzugucken ob es geht?

**Comment by Trunks1982** [comment]
Super Danke.
